libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -701,12 +697,12 @@ constructor(
          * @return default [NavigationOptions]
          */
         @JvmStatic
-        fun defaultNavigationOptions(context: Context, accessToken: String?): NavigationOptions {
+        fun defaultNavigationOptions(context: Context, accessToken: String?): NavigationOptions.Builder {
```

Comment:
NIT: since we are returning a `Builder` instead of the `Options` itself, should we update fun name as well?